### PR TITLE
#150: Deleted entry on resource localized for pt and fixed test

### PR DESCRIPTION
**Pull request by magnonasc**
I couldn't reopen #151 after those end of line problems, so I'm open this one.

### CODE REVIEW

File: src/test/java/tec/uom/se/format/SymbolMapTest.java [hunk 1]
```diff
@@ -52,7 +52,7 @@
    */
   @Test
   public void testSymbolMap() {
-    SymbolMap symbols = SymbolMap.of(ResourceBundle.getBundle(BUNDLE_NAME, Locale.getDefault()));
+    SymbolMap symbols = SymbolMap.of(ResourceBundle.getBundle(BUNDLE_NAME, Locale.ROOT));
```
Comment (magnonasc) [inline src/test/java/tec/uom/se/format/SymbolMapTest.java]: The biggest issue here is that you were using `Locale.getDefault()` to test `messages.properties`, but `getDefault()` means "give me the default locale being used by the current machine", and not a "root" locale that would get the default `messages.properties`.

Comment (garretwilson) [inline src/test/java/tec/uom/se/format/SymbolMapTest.java]: Ah, that's a very good use of `Locale.ROOT`. You're basically saying not to use locale-specific overrides --- you just want to use the symbols in the base resource bundles.

Comment (magnonasc) [inline src/test/java/tec/uom/se/format/SymbolMapTest.java]: Yeeah, and it won't make the test fail if the user tries to run the tests on Japan or some of these other countries where there's a different symbol for some units :D

**Comment by magnonasc** [comment]
I didn't delete the pt resource, just that entry, which wasn't making any sense at all. Any specific reason for having `mu`? instead of the symbol itself?

**Comment by keilw** [comment]
It was translated by a Native speaker, seems the Portuguese do not use the symbol.

**Comment by magnonasc** [comment]
see https://www.healthcare.siemens.pt/computed-tomography/technologies-innovations/ufc-ultra-fast-ceramic/technology and https://academic.ieee.org/docs/4u as example websites from Portugal, I think the person didn't know what he was doing :/

**Pull request merged**